packages/shared/lib/integrations/scripts/webhook/webhook.manager.ts

```diff
@@ -10,7 +11,7 @@ import type { WebhookHandlersMap, WebhookResponse } from './types.js';
 
 const handlers: WebhookHandlersMap = webhookHandlers as unknown as WebhookHandlersMap;
 
-async function execute(environmentUuid: string, providerConfigKey: string, headers: Record<string, any>, body: any, rawBody: string): Promise<void | any> {
+async function execute(environmentUuid: string, providerConfigKey: string, headers: Record<string, any>, body: any, rawBody: string, span: Span): Promise<any> {
```

Comment:
this is probably nitpicking but a span is usually a function call or a logical unit. Here the the api call will be marked as error instead of the processing of the webhook.